Factorial

```diff
@@ -8,7 +8,7 @@ int main()
     if (num >= 0)
     {
         for (long i = 2; i <= num; i++)
-        factorial *= i; //factorial = factorial * i;
+        factorial = factorial * i;
```

Comment:
Create a function to calculate factorial

Reply:
#include <iostream>

int factorial(int n)
{ 
    int fact = 1;
    if (n==0 || n==1)
    return (1);
    else
    for (int i = 1; i <= n; ++i)
    fact *= i;
    return (fact);
}
int main ()
{ 
    int n;
    std::cin >> n;
    std::cout << factorial(n);
}

Reply:
Չհասկացա. առաջին տողում <iostream>-ը ջնջվեց((

Reply:
Էլի չգրվեց, ինչու՞